src/Header/Header.js

```diff
@@ -24,7 +24,9 @@ const Header = (props) => {
           <button data-testid={'login'} onClick={props.displayLoginPage}>Login</button>
         </nav>
       }
-      <p>{userDisplay}</p>
```

Comment:
It might be worth your time taking a crack at implementing React Router here! The components being rendered here are perfect components to add routing to! Use the lesson plan as a reference and do some pre-teaching for tomorrow!